### DISCUSSION TITLE
Add eject icon

### DISCUSSION THREAD

**Discussion by rohitgta7** [tailwindlabs/heroicons#1068] [Ideas]
Please add the eject icon

**Comment by jcandan** [comment]
An eject icon features a triangle with a rectangle below it.

Consider the following?

---

```html
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
  <path stroke-linecap="round" stroke-linejoin="round" d="M12 6.75 7.5 12.75h9L12 6.75Z"></path>
  <path stroke-linecap="round" stroke-linejoin="round" d="M7.5 16.5h9"></path>
</svg>
```

https://www.svgviewer.dev/s/0iL08xS1

<img width="53" height="53" alt="image" src="https://github.com/user-attachments/assets/2952aba1-b6e2-4444-8eb7-f51bf0f51be1" />

---

```html
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
  <path stroke-linecap="round" stroke-linejoin="round" d="M12 6.75 7.5 12.75h9L12 6.75Z"/>
  <rect x="7.5" y="16" width="9" height="1.75" rx=".5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>
```

https://www.svgviewer.dev/s/I1bVLFRM

<img width="56" height="55" alt="image" src="https://github.com/user-attachments/assets/26d74a85-f677-4c7d-8e0f-b9047073871c" />
